zScanner/Views/NewDocument/PhotoSelectorCollectionViewCell.swift

```diff
@@ -103,7 +104,7 @@ class PhotoSelectorCollectionViewCell: UICollectionViewCell {
 
 extension PhotoSelectorCollectionViewCell: UITextFieldDelegate {
     func textFieldDidBeginEditing(_ textField: UITextField) {
-        textField.setBottomBorder(color: UIColor.red.cgColor, animated: true, duration: 0.6)
```

Comment:
Don't use system color `.red` here. We have some predefined colors in `UIColor.swift` file in extensions :) Take a look there.